financepayment/classes/divido.class.php

```diff
@@ -202,9 +203,9 @@ public function getCartPlans($cart)
     public function getPlans($default_plans = false)
     {
         if ($default_plans) {
-            $plans = $this->getGlobalSelectedPlans();
+            return $this->getGlobalSelectedPlans();
         } else {
```

Comment:
```suggestion
        } 
        
        return $this->geAllPlans();
        }
```
It's always going to hit the if or the else, so you might as well drop the else and just return `$this->getAllPlans();` rather than `$plans`, which doesn't get set anyway